recipe/meta.yaml

```diff
@@ -28,18 +27,17 @@ requirements:
   host:
     - pip
     - python
-    - setuptools >=30.3.0
+    - setuptools >=42
     - setuptools_scm
-    - setuptools_scm_git_archive
-    - numpy
+    - numpy >=2
     - fontconfig  # [unix]
-    - cython >=0.29.2
+    - cython >=3.0.0
     - packaging
     - wheel
 
   run:
     - python
-    - {{ pin_compatible('numpy') }}
+    - numpy
     - fontconfig  # [unix]
```

Comment:
just on unix? The patch could be made to work on win-64 too perhaps

Reply:
This is how the package was before so this wouldn't be a regression, in terms of priorities I think this is lower than others.